mne/time_frequency/_stockwell.py

```diff
@@ -207,7 +197,14 @@ def tfr_array_stockwell(data, sfreq, fmin=None, fmax=None, n_fft=None,
 def tfr_stockwell(inst, fmin=None, fmax=None, n_fft=None,
                   width=1.0, decim=1, return_itc=False, n_jobs=1,
                   verbose=None):
-    """Time-Frequency Representation (TFR) using Stockwell Transform.
+    """Compute Time-Frequency Representation (TFR) using Stockwell Transform.
+
+    Same computation as the function tfr_array_stockwell.
+    Difference: tfr_stockwell takes epochs in the mne.Epochs format as input;
+    tfr_array_stockwell takes epochs in the numpy.array format as input.
```

Comment:
```suggestion
    Same computation as `~mne.time_frequency.tfr_array_stockwell`, but operates
    on `~mne.Epochs` objects instead of :class:`NumPy arrays <numpy.ndarray>`.
```